modules/components/pages/redpanda/about.adoc

```diff
@@ -23,15 +50,27 @@ redpanda:
     root_cas: ""
     root_cas_file: ""
     client_certs: []
-  sasl: []
-----
+  sasl: [] # No default (optional)
+```
 
+--
+======
 
 == Fields
 
```

Comment:
Are any of these fields cloud only?

Reply:
Most of these fields are automatically filled in with cloud deployments. It's not strictly disallowed, but we wouldn't expect a user to configure any of this stuff in a cloud environment.

Reply:
@Jeffail Does a description of this component need to appear in the Cloud docs at all? It doesn't at the moment. Or are you just saying that a description of the component is all that is required for Cloud stating that it is automatically configured for users?

Reply:
It's sort of a hidden feature for cloud users because it's how the logs appear in the UI. It's possible for users to customise the redpanda section but it will be overwritten when deployed, so other than maybe mentioning that it's redundant in a cloud environment we don't really need to expand on it just yet.

We're still figuring out how this section will interact with other components that are on the roadmap and depending on how that design plays out we might want to document this section in more detail on cloud but for now I'd say keep it minimal.

Reply:
OK, I think if we can't describe how Cloud users will use this feature we shouldn't introduce it to the Cloud docs. Do you agree @Jeffail?